new-lamassu-admin/src/pages/Triggers/CustomInfoRequests/CustomInfoRequests.js

```diff
@@ -30,44 +33,110 @@ const constraintTypeDisplay = {
   spaceSeparation: 'Space separation'
 }
 
+const ADD_ROW = gql`
+  mutation insertCustomInfoRequest($customRequest: CustomRequestInput!) {
+    insertCustomInfoRequest(customRequest: $customRequest) {
+      id
+    }
+  }
+`
+const EDIT_ROW = gql`
+  mutation editCustomInfoRequest(
+    $id: ID!
+    $customRequest: CustomRequestInput!
+  ) {
+    editCustomInfoRequest(id: $id, customRequest: $customRequest) {
+      id
+    }
+  }
+`
+
+const REMOVE_ROW = gql`
+  mutation removeCustomInfoRequest($id: ID!) {
+    removeCustomInfoRequest(id: $id) {
+      id
+    }
+  }
+`
+
 const CustomInfoRequests = ({
-  customRequests,
   showWizard,
   toggleWizard,
-  handleDelete,
-  handleSave
+  data: customRequests
 }) => {
   const classes = useStyles()
+
   const [toBeDeleted, setToBeDeleted] = useState()
   const [toBeEdited, setToBeEdited] = useState()
   const [deleteDialog, setDeleteDialog] = useState(false)
+
+  const [addEntry] = useMutation(ADD_ROW, {
+    onError: () => console.log('Error while adding custom info request'),
+    refetchQueries: () => ['customInfoRequests']
+  })
+
+  const [editEntry] = useMutation(EDIT_ROW, {
+    onError: () => console.log('Error while editing custom info request'),
+    refetchQueries: () => ['customInfoRequests']
+  })
+
+  const [removeEntry] = useMutation(REMOVE_ROW, {
+    onError: () => console.log('Error while removing custom info request'),
+    refetchQueries: () => ['customInfoRequests']
+  })
```

Comment:
Should these errors output something meaningful to the user, such as 'Internal server error' or something? Better to confirm this with Rafael. There are some examples of this kind of error handling throughout the code.